api/db/database-builder/factory/build-complementary-certification.js

```diff
@@ -24,4 +24,28 @@ module.exports = function buildComplementaryCertification({
     tableName: 'complementary-certifications',
     values,
   });
+}
+
+buildComplementaryCertification.clea = function ({
+  id = databaseBuffer.getNextId(),
+  label = 'CléA Numérique',
+  key = 'CLEA',
+  createdAt = new Date('2020-01-01'),
+  minimumReproducibilityRate = 50.0,
+  minimumEarnedPix = 70,
+  hasComplementaryReferential = false,
+  hasExternalJury = false,
+}) {
+  return buildComplementaryCertification({
+    id,
+    label,
+    key,
+    createdAt,
+    minimumReproducibilityRate,
+    minimumEarnedPix,
+    hasComplementaryReferential,
+    hasExternalJury,
+  });
 };
```

Comment:
- Cette methode ne doit pas prendre en argument les specificités CLEA. 
- La `key` peut-etre importé

```suggestion
  createdAt = new Date('2020-01-01'),
  minimumReproducibilityRate = 50.0,
  minimumEarnedPix = 70,
}) {
  return buildComplementaryCertification({
    id,
    label: 'CléA Numérique',
    key: CertificationCourse.CLEA,
    createdAt,
    minimumReproducibilityRate,
    minimumEarnedPix,
    hasComplementaryReferential:false,
    hasExternalJury: false,
  });
};
```